polyply/src/gen_itp.py

```diff
@@ -70,9 +74,15 @@ def gen_itp(args):
          raise IOError("Cannot parse file with extension {}.".format(extension))
 
     # Do transformationa and apply link
-    meta_molecule = MapToMolecule().run_molecule(meta_molecule)
+    meta_molecule = MapToMolecule(force_field).run_molecule(meta_molecule)
     meta_molecule = ApplyLinks().run_molecule(meta_molecule)
 
+    # Raise warning if molecule is disconnected
+    if not nx.is_connected(meta_molecule.molecule):
+        for component in nx.connected_components(meta_molecule.molecule):
+            print(component)
```

Comment:
Shouldn't this be part of the logged warning instead?

Reply:
yes; and in fact I moved this to PR #123 as that implements the Logger